apps/backoffice/src/sections/auth/components/RegisterForm.tsx

```
@@ -0,0 +1,132 @@
+
+  return (
+    <div className="space-y-6 w-full max-w-md">
+      <div className="space-y-2 text-center">
+        <h1 className="text-3xl font-bold">Crear cuenta</h1>
+        <p className="text-gray-500">Ingrese sus datos para registrarse</p>
+      </div>
+
+      {(error ?? authError) && (
+        <div className="bg-destructive/15 p-3 rounded-md text-destructive text-sm">
+          {error ?? (authError && authError.detail)}
+        </div>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)